src/tests/boss/RegressionForecaster.test.js

```diff
@@ -155,6 +156,7 @@ describe('RegressionForecastingRule', function() {
 
          forecaster._addEmptySlotsToTimeSeries(series, 0);
 
+
```

Comment:
![](https://i.imgflip.com/2i7zoh.jpg)

Reply:
haha, kind of ironic in the test `adds no slots to empty series`